pkg/controller/datavolume-controller.go

```diff
@@ -438,6 +438,20 @@ func (r *DatavolumeReconciler) Reconcile(_ context.Context, req reconcile.Reques
 		return reconcile.Result{}, err
 	}
 
+	// When cloning a PVC, if the target's size is not specified,
+	// said value can be attainable from the source PVC
+	if selectedCloneStrategy != NoClone {
+		targetRequest, hasTargetRequest := pvcSpec.Resources.Requests[corev1.ResourceStorage]
```

Comment:
if I understand correctly at this point hasTargetRequest can't be false right? maybe return an error if somehow it is?

Reply:
Yes, It shouldn't be, but I wanted to make sure. I'll later add proper error handling, thanks!